src/test/java/dev/jbang/cli/TestJdk.java

```diff
@@ -32,9 +34,7 @@ void testNoJdksInstalled() throws IOException {
 
 	@Test
 	void testHasJdksInstalled() throws IOException {
-
-		final File testCache = initJBangCacheDir();
-		final File jdkPath = new File(testCache, "jdks");
+		final File jdkPath = new File(jbangTempCacheDir.toFile(), "jdks");
```

Comment:
Use `JdkManager.getJdksPath()`

Reply:
Hehe, that is a private method. I can make it public to make it work.

Reply:
Yeah, probably okay to do that 👍